### PR TITLE
Add arm64 build

### PR DESCRIPTION
**Pull request by povilasv**
Adds drone.io arm64 build

Also fixes `strconv.Atoi(fromString)` on 32 bit systems returns ant int32, casting to `int64(from)` then looses values and doesn't work

Ref https://github.com/conprof/conprof/issues/44

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -47,3 +47,97 @@ steps:
...
+name: build-arm64
+
+platform:
+  os: linux
+  arch: arm64
+
+steps:
+- name: build
+  pull: always
+  image: golang:1.13
```
Comment (brancz) [inline .drone.yml:61]: Any reason not to use 1.14?

File: .drone.yml [hunk 2]
```diff
@@ -47,3 +47,97 @@ steps:
...
+    branch:
+    - master
+    event:
+    - push
+---
+kind: pipeline
+name: manifest
+
+platform:
+ os: linux
```
[the remark this answers is not in the document]
Comment (povilasv) [reply .drone.yml]: Fixed

File: .drone.yml (hunk 1 above)
Comment (povilasv) [reply .drone.yml]: just copy paste from old version of drone.yaml

**Comment by povilasv** [comment]
I've just test 1.14, all good everything seems to work :)

**Comment by brancz** [comment]
Thanks! :tada:

**Pull request merged**